src/Microsoft.DotNet.Interactive/Connection/ProxyKernel.cs

```diff
@@ -34,24 +34,22 @@ public sealed class ProxyKernel : Kernel
 
         if (remoteUri is not null)
         {
-            _remoteUri = remoteUri;
+            KernelInfo.RemoteUri = remoteUri;
         }
         else if (sender.RemoteHostUri is { } remoteHostUri)
         {
-            _remoteUri = new(remoteHostUri, name);
+            KernelInfo.RemoteUri = new(remoteHostUri, name);
```

Comment:
I wonder if the bug we were looking at in the browser kernel yesterday means this the wrong behavior. It sort of works by accident, sometimes, but it's not really a safe assumption.